controllers/defaults/placements.go

```diff
@@ -176,6 +176,25 @@ func getWeightedPodAffinityTerm(weight int32, selectorValue ...string) corev1.We
 	return WeightedPodAffinityTerm
 }
 
+func getMdsWeightedPodAffinityTerm(weight int32, selectorValue ...string) corev1.WeightedPodAffinityTerm {
+	WeightedPodAffinityTerm := corev1.WeightedPodAffinityTerm{
+		Weight: weight,
+		PodAffinityTerm: corev1.PodAffinityTerm{
+			LabelSelector: &metav1.LabelSelector{
+				MatchExpressions: []metav1.LabelSelectorRequirement{
+					{
+						Key:      "app.kubernetes.io/part-of",
```

Comment:
This label seems more clear for the intention that we want antiaffinity to the same filesystem
```suggestion
						Key:      "rook_file_system",
```